doc/shapes/2025.02/add_support_for_library_publication.adoc

```diff
@@ -0,0 +1,72 @@
+= Add support for library publication
+
+== Problem
+
+SysON doesn't provide an  **in-application** way to import models from other projects.
+The current import/export mechanisms rely on generating a textual SysML file, and uploading such file in a project to import it.
+
+When exported models are used as shared reusable components, it is the responsibility of the stakeholders to:
+- Store the shared models in an external location
+- Ensure that every user is working on the same version of the shared model
+- Coordinate to export a new version when needed, and import it in all the dependent projects
+
+These processes are error-prone, and could be more controlled if they all happened inside the application.
+
+== Key Result
+
+SysON users should be able to:
+
+- Publish a library with a given version number and description
+- See published libraries
+
+== Solution
+
+The command palette provides a command to publish libraries.
+This command opens a dialog asking the user to provide the version and the description of the library they want to publish.
+
+The publication process creates a single, self-contained library containing the entire project.
+Standard libraries are not published as part of the library (we assume they are always available and don't need to be stored in libraries).
+
+It is not possible to publish a library with the same projectId/name/version as an existing library (this constraint is enforced by Sirius Web).
+
```

Comment:
Could you add in the `Solution` section a paragraph about how to see the list of published libraries?